src/components/styledSelect/styledSelect.module.css

```diff
@@ -0,0 +1,84 @@
+.root {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  background-color: #fff;
+  font-size: 1rem;
+  margin: auto 0;
+  width: 228px;
+  position: relative;
+}
+
+.disabled {
+  cursor: not-allowed;
+}
+
+.header {
+  border: 1px solid #5f5f5f;
+  border-radius: 3px;
+  display: flex;
+  justify-content: space-between;
+  grid-template-columns: 8fr 1fr;
+}
+
+.headerText {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  color: #5f5f5f;
+  padding: 0 8px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  height: 1rem;
+  line-height: 1;
```

Comment:
When this is removed, the text appears towards the bottom of the containing element instead of centred vertically.